inference-engine/src/vpu/graph_transformer/src/stages/rnn.cpp

```diff
@@ -102,18 +102,21 @@ static void RNNRelayout(
 
                  fp16_t* dst0,
                  fp16_t* dst1,
+                 fp16_t* dst2,
```

Comment:
why do you change this method? Passing number of outputs to Blob means that you should add one more parameter to the serializer.

Reply:
like `serializer.append(static_cast<int>(useCellState))`